crypto/rand/rand_win.c

```diff
@@ -71,6 +71,8 @@ int RAND_poll_ex(RAND_poll_fn cb, void *arg)
             ok++;
```

Comment:
yes because it's checked before returning.  I could make it "ok = 1" if that's more clear.